src/main/java/com/example/booksstore/controller/BookController.java

```diff
@@ -16,32 +20,40 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "/api/books")
 public class BookController {
     private final BookService bookService;
 
+    @Operation(summary = "Get all books in shop",
+            description = "Get a list of all available books")
```

Comment:
You get not all if you use pageable.

Reply:
you can just say "returns a page of books"